docker/README.md

```diff
@@ -1,632 +1,103 @@
-includes repositories in GitHub, you need to specify your GitHub
-token overriding the `/override.cfg` file with `mytoken.cfg`,
-with the following content (XXX should be substituted by a real token):
-
-```
-[github]
-api-token = XXX
-```
-
-And it will be run as:
-
-
-```bash
-$ docker run --net="host" \
-    -v $(pwd)/mytoken.cfg:/override.cfg \
-    grimoirelab/installed
-```
-
-It is convenient to see the logs written by mordred while
-producing the dashboard. Those are produced within the container,
-in the `/logs` directory. To make them available in the host,
-just map that directory to a host directory:
-
-```bash
-$ docker run --net="host" \
-    -v $(pwd)/mytoken.cfg:/override.cfg \
-    -v $(pwd)/logs:/logs \
-    grimoirelab/installed
-```
-
-If the default configuration is not appropriate, for example because
-the database ports or credentials are not the ones used,
-the infrastructure configuration file can be overridden:
-
-```bash
-$ docker run --net="host" \
-    -v $(pwd)/mytoken.cfg:/override.cfg \
-    -v $(pwd)/logs:/logs \
-    -v $(pwd)/infra-local.cfg:/infra.cfg \
-    grimoirelab/installed
-```
-
-If you also want to override the list of repositories to analyze,
-you need to map the `/projects.json` file in the container:
-
-```bash
-$ docker run --net="host" \
-    -v $(pwd)/mytoken.cfg:/override.cfg \
-    -v $(pwd)/logs:/logs \
-    -v $(pwd)/infra-local.cfg:/infra.cfg \
-    -v $(pwd)/myprojects.json:/projects.json \
-    grimoirelab/installed
+To produce the image, run the following command:
 ```
-
-You can also override the `menu.yaml` file,
-to have a custom menu for your dashboard:
-
-```bash
-$ docker run --net="host" \
-    -v $(pwd)/mytoken.cfg:/override.cfg \
-    -v $(pwd)/logs:/logs \
-    -v $(pwd)/infra-local.cfg:/infra.cfg \
-    -v $(pwd)/myprojects.json:/projects.json \
-    -v $(pwd)/mymenu.yaml:/menu.yaml \
-    grimoirelab/installed
+$ docker build -f Dockerfile -t grimoirelab/grimoirelab .
 ```
 
-## grimoirelab/full
-
-`grimoirelab/full` is a container based on `grimoirelab/installed`,
-which therefore includes GrimoireLab, but also all the services
-needed to produce a GrimoireLab dashboard: Elasticsearch, MariaDB,
-and Kibana. As is, it will produce a dashboard for the GrimoireLab
-project.
-
-To try it, you can just run it as follows:
-
-```bash
-$ docker run -p 127.0.0.1:5601:5601 \
-    -v $(pwd)/mytoken.cfg:/override.cfg \
-    -t grimoirelab/full
-```
+You can build a specific version of grimoirelab updating `GRIMOIRELAB_RELEASE`
+environment variable inside the Dockerfile.
 
-`myytoken.cfg` should have a GitHub API token, in a `mordred.cfg`
-format, as explained above. Let's repeat it here:
+## Run the image
 
+You can run any GrimoireLab tool using this image, for example, you can collect 
+Git commits using Perceval:
 ```
-[github]
-api-token = XXX
+$ docker run grimoirelab/grimoirelab perceval git https://github.com/chaoss/grimoirelab-perceval.git
```

Comment:
This should go after the next paragraph because it's not the default use. I would rephrase it as `You can also run ...`